contracts/external/cw-payroll/src/contract.rs

```diff
@@ -80,59 +182,96 @@ pub fn execute_create_stream(
 
     let block_time = env.block.time.seconds();
     // TODO: Change to support previous dates, within given reason?
-    //       The problem with this is it restricts backdating, 
+    //       The problem with this is it restricts backdating,
     //       so all funds must vest before distribute (for good or bad)
     // TODO: start_time Could default to now?
-    if start_time <= block_time {
+    // UPD: Fixed
+    if start_time <= block_time && end_time <= block_time {
```

Comment:
remove this if block. start time could be `0`, because we want to allow starting to occur in past, if say compensation was started in july 2022 but the stream was created in november 2022 and ends in july 2023.